### PR TITLE
add G7 EA JD

### PR DESCRIPTION
**Pull request by MercerT77**

### CODE REVIEW

File: Architects/G7 Enterprise Architect.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+-   Helping to mature architecture practice and profession to ensure what we deliver is strategically aligned and provides best value. Helping to maintain the architecture body of knowledge, developing standards and patterns that contribute back to wider technical communities.
+    
+-   Providing leadership and support in the architecture community (which may include line management), offering independent leadership to ensure issues are resolved positively across DfE.
+    
+-   Supporting the architecture community in DfE, ensuring a diverse group of people feel welcome, respected and supported, as well as supporting your counterparts in other parts of government in building an active community that everyone in architecture and leadership across government can belong to and participate in.
+    
+
+  
+  
+
```
Comment (luke-slowen) [inline Architects/G7 Enterprise Architect.md:52]: Remove extra line breaks

File: Architects/G7 Enterprise Architect.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
+# Enterprise Architect
+
```
Comment (luke-slowen) [inline Architects/G7 Enterprise Architect.md:2]: Did we decide against adding a summary, like TA?

File: Architects/G7 Enterprise Architect.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+You’ll​ ​undertake​ ​structured​ ​analysis​ ​of​ ​technical​ ​issues,​ ​translating​ ​this​ ​analysis​ ​into ​designs​ ​that​ ​describe​ ​a​ ​solution. ​You​ ​can​ ​be​ ​consulted​ ​on​ ​design​ ​and​ ​are capable​ ​of​ ​providing​ ​design​ ​patterns. ​ ​You​ ​can​ ​identify​ ​the​ ​deeper​ ​issues​ ​that​ ​need fixing​ ​and​ ​look​ ​for​ ​opportunities​ ​to​ ​collaborate​ ​and​ ​reuse​ ​components,​ ​communicating with​ ​both​ ​technical​ ​and​ ​non-technical​ ​stakeholders. Collaborating with other DfE leaders, you will help build an architecture capability that can deliver the DfE priorities, as well as supporting capability building across government.
+
+  
+
+You will have oversight of the work of architecture community members on projects, programmes and business as usual activities, ensuring work is of an appropriate standard. Where appropriate, you should lead by example and positively resolve issues collaboratively.
+
+There will be opportunities to contribute to the development of the pan-government community and wider technology leadership across government.
+
+### Key responsibilities
```
Comment (luke-slowen) [inline Architects/G7 Enterprise Architect.md:25]: Should be an H2 ##

File: Architects/G7 Enterprise Architect.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+    
+-   Architecture qualifications or certifications (such as TOGAF, Business Architecture Guild)
+    
+-   Line management experience, including coaching, mentoring and developing yourself & others
+    
+
+  
+  
+
+### Selection process details
```
Comment (luke-slowen) [inline Architects/G7 Enterprise Architect.md:100]: H2 ##

File: Architects/G7 Enterprise Architect.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+  
+
+#### Application
```
Comment (luke-slowen) [inline Architects/G7 Enterprise Architect.md:104]: H3 ###, same for following headings

**Review by luke-slowen — CHANGES_REQUESTED** [review]
Just minor formatting tweaks, as per in-line comments. Could we also make it a generic EA page title please? (drop the grade)

**Comment by MercerT77** [comment]
Ok, so a couple of things……

I can’t remember us talking about putting in a summary like the TA role – we only agreed on the selection process to mirror the TA role, I think.

Do you want ‘The Role’ section replacing with the ‘Summary’?

Also, I’m not entirely sure how to amend the file in Github……it’s been a while……I could ask Lewis



Theresa Mercer
Architecture Profession Manager

Digital & Technology
Operations Group

M: 07443 292508 E: ***@***.******@***.***>
A: Cheylesmore House, 5 Quinton Road, CV1 2WT
W: http://www.gov.uk/dfe

Advanced notice of my Holiday Dates:


***@***.***

***@***.***[AgilePM+Practitioner-01+_281_29]

**Comment by MercerT77** [comment]
MERCER, Theresa would like to recall the message, "[DFE-Digital/dfe-job-descriptions] add G7 EA JD (PR #17)".

**Comment by MercerT77** [comment]
Have made the amendments. Have a look to make sure you're happy.

**Comment by MercerT77** [comment]
Selection process amended to H2

**Review by luke-slowen — APPROVED** [review]
All good, thanks!

**Pull request merged**